test/e2e/lib/jest/globalSetup.ts

```diff
@@ -0,0 +1,47 @@
+import { mkdir } from 'fs/promises';
+import path from 'path';
+import config from 'config';
+import { chromium } from 'playwright';
```

Comment:
I looked at importing `@automattic/calypso-e2e`, however the [jest-conditionals.ts](https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-e2e/src/jest-conditionals.ts) file triggers the error `Do not import `@jest/globals` outside of the Jest test environment`.